x-pack/plugins/data_usage/public/app/components/data_usage_metrics.tsx

```diff
@@ -118,10 +123,16 @@ export const DataUsageMetrics = memo(
       },
       {
         retry: false,
-        enabled: !!metricsFilters.dataStreams.length,
+        enabled: !!(metricsFilters.dataStreams.length && metricsFilters.metricTypes.length),
```

Comment:
updated the network request logic to work only if there are metric types and data streams.